CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
-## master
```

Comment:
Mention the breaking change to the RouteController initializer.

Reply:
Heh, maybe I should’ve mentioned this for #1671 while I had the chance. 😅

Reply:
👍 will address